test/map_matchers_test.dart

```diff
@@ -53,5 +54,34 @@ void main() {
       "Actual: {'a': 1, 'null': null} "
       "Which:  doesn't contain key '2'",
     );
+    shouldFail(
+      {null: null},
+      containsPair('2', 'b'),
+      "Expected: contains pair '2' => 'b' "
+      'Actual: {null: null} '
+      "Which:  doesn't contain key '2'",
+    );
```

Comment:
Remove this case, it's covered by lines 79-85 with more descriptive values.

Reply:
ack